src/app/static/src/app/store/translations/locales.ts

```diff
@@ -161,7 +161,7 @@ export interface Translations {
     missingError: string,
     modelOptions: string,
     newPassword: string,
-    next: string,
+    nextPage: string,
```

Comment:
I don't believe we need `next` any more - I think this was added when the Data Exploration component was being delveloped, but then 'next' button was removed and replaced by StepperNavigation